airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -67,6 +67,31 @@
         "maximum": 1095,
         "default": 14,
         "examples": [14]
+      },
+      "custom_query": {
+        "type": "array",
+        "items": {
+          "type": ["object", "null"],
+          "properties": {
+            "query": {
+              "type": "string",
+              "title": "User defined query to build a report by wish",
+              "examples": ["SELECT segments.ad_destination_type FROM campaign"]
+            },
+            "primary_key": {
+              "type": ["string", "null"],
+              "title": "The unique field to be used as primary key in destination database (if provided)"
+            },
+            "cursor_field": {
+              "type": ["string", "null"],
+              "title": "If not provided, the FULL-REFRESH model will be used. If provided, this will be an incremental stream with this cursor field. Please use datetime fields (start_date, end_date) for proper work."
```

Comment:
```suggestion
              "title": "Field that will be used to track changes during Incremental sync (if provided). Otherwise only Full Refresh sync will be available."
```